### DISCUSSION TITLE
Extract date from holyday

### DISCUSSION THREAD

**Discussion by Rembardino** [vacanza/holidays#850] [Q&A] [answered]
How do I extract the date of the desired holiday...for example, i want to know the Thankgiving date of year 2002, how do I do that?

**Comment by arkid15r** [comment, marked as answer]
You could use [HolidayBase::get_named()](https://python-holidays.readthedocs.io/en/latest/api.html#holidays.holiday_base.HolidayBase.get_named) for that
```
>>> import holidays
>>> us_holidays = holidays.US(years=2022)
>>> us_holidays.get_named("Thanksgiving")
[datetime.date(2022, 11, 24)]
```
